### DISCUSSION TITLE
Performance testing

### DISCUSSION THREAD

**Discussion by slobodadev** [vercel/vercel#6831] [Help]
Hi! Would you be so kindly to help with performance testing? 
Our team have problem with performance testing. We've tried to run performance tests on our Next.js app at Vercel and one of our teammates was blocked. How can we run performance tests without blocks? How can we unblock our developer? Can we have some whitelisted ip addresses for tests or temporary disable restrictions? How can we estimate our load limitations? For example, can we handle million users? How to calculate price if it is possible?

**Comment by leerob** [comment]
👋 Please contact sales to discuss load testing -> https://vercel.com/contact/sales
